tests/integration/security/mtls_first_party_jwt/strict_test.go

```diff
@@ -32,7 +32,7 @@ import (
 // - Send HTTP/gRPC requests between apps.
 func TestMtlsStrictK8sCA(t *testing.T) {
 	framework.NewTest(t).
-		Features("security.control-plane.k8s-certs").
+		Features("security.control-plane.k8s-certs.k8sca.first-party-jwt").
```

Comment:
the other one is named `security.control-plane.k8s-certs.k8sca.kubernetes` which is very confusing when compare with this `first-party-jwt`

Reply:
change this one to security.control-plane.k8s-certs.jwt on mtlsfirstpartyjwt/TestMtlsStrictK8sCA
change this one to security.control-plane.k8s-certs.k8sca on mtlsk8sca/TestMtlsStrictK8sCA